cves/kernel/CVE-2016-0823.yml

```diff
@@ -371,8 +382,8 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: The vulnerability had to add checks without moving anything else.
```

Comment:
Is this not an order of operations? It changed the order by adding a new operation.

Reply:
I think it's not an order of operations problem because there was no code that was previously present that they had to change the order of.